inputs/modules/external_coupling/interconnectors/external_coupling_import_interconnector_6.ad

```diff
@@ -0,0 +1,19 @@
+- query =
+  IF(
+    EQUALS(USER_INPUT(), 1.0),
+    -> {
+      UPDATE(
+        UPDATE(V(energy_interconnector_6_imported_electricity, merit_order), subtype, must_run);
+        UPDATE(V(energy_interconnector_6_imported_electricity, merit_order), group, interconnector_6_import_must_run);
+      );
```

Comment:
```suggestion
    -> {
      UPDATE(
        UPDATE(V(energy_interconnector_6_imported_electricity, merit_order), type, producer);
        UPDATE(V(energy_interconnector_6_imported_electricity, merit_order), subtype, must_run);
        UPDATE(V(energy_interconnector_6_imported_electricity, merit_order), group, interconnector_6_import_must_run);
      );
```

Their type is `flex` from the node file, you need to change in to `producer` for producers.

Reply:
Thanks! I've made the change now.